### ISSUE TITLE
Only show currently active option for preview

### ISSUE THREAD

**Issue by jmarolf** [dotnet/roslyn#14911]
**Version Used**:  d15prerel/15.0.25831.2

Expression preferences, Variable preferences, and 'null' checking show before and after in same preview.

![image](https://cloud.githubusercontent.com/assets/9797472/19947792/f8727b52-a106-11e6-936a-f7511b15377d.png)

For all other options in Code Style we show only the currently active option
![image](https://cloud.githubusercontent.com/assets/9797472/19947824/1139d5a4-a107-11e6-848d-800d5bd8af84.png)

**Comment by kuhlenh** [comment]
I noticed that @CyrusNajmabadi did this and I actually like it better. 

From the UX studies we did on this UI, many folks were confused on what the preview was showing and many folks read the rules *backwards*, i.e., instead of reading it as "Prefer the explicit type for built-in types otherwise report violations as a Warning", they interpreted it as "If you see an explicit type instead of var for a built-in type, report a Warning". Showing the "before" and "after" would aid in the comprehension of this UI. 

Thoughts? @jmarolf @Pilchie
